Domain/ConfigurationExtensions.cs

```diff
@@ -100,6 +100,14 @@ public static ICommandScheduler<TAggregate> CommandScheduler<TAggregate>(this Co
         internal static ISnapshotRepository SnapshotRepository(this Configuration configuration)
         {
             return configuration.Container.Resolve<ISnapshotRepository>();
-        }
+        }
+
+        /// <summary>
+        /// Gets an <see cref="IReservationService"/>
+        /// </summary>
+        public static IReservationService ReservationService(this Configuration configuration)
+        {

```

Comment:
There's already a `Configuration.ReservationService` property that effectively does this same thing (but it's also settable and has a fallback). Is this extension method needed?